examples/ota-requestor-app/ameba/main/chipinterface.cpp

```diff
@@ -136,18 +146,7 @@ extern "C" void ChipTest(void)
         ChipLogProgress(DeviceLayer, "DeviceManagerInit() - OK\r\n");
     }
 
-    chip::Server::GetInstance().Init();
-
-    // Initialize device attestation config
-    SetDeviceAttestationCredentialsProvider(Examples::GetExampleDACProvider());
-    NetWorkCommissioningInstInit();
+    chip::DeviceLayer::PlatformMgr().ScheduleWork(InitServer, reinterpret_cast<intptr_t>(nullptr));
 
     InitOTARequestor();
```

Comment:
Shouldn't InitOTARequestor be happening in InitServer?

Reply:
This has been fixed.